webclient/core/ident_cely.py

```diff
@@ -336,17 +281,32 @@ def get_temp_lokalita_ident(typ, region, lokalita):
     """
     Metóda pro výpočet dočasného identu lokality.
 
-    Logika složení je: "X-" + region (M anebo C) + "-" + typ + 7 místne číslo (id z tabulky lokality doplněno na 7 čísel nulama)
-    Příklad: "X-M-L0001234"
+    Logika složení je: "X-" + region (M anebo C) + "-" + typ + 9 místne číslo ze sekvence lokalita_xident_seq doplňeno na 9 číslic.
+    
+    Příklad: "X-M-L000123456"
     """
-    MAXIMAL: int = 9999999
-    # [region] - [typ] - [7 mistne cislo]
     prefix = str(IDENTIFIKATOR_DOCASNY_PREFIX + region + "-" + typ)
-    if lokalita.id is not None:
-        id_number = "{0}".format(str(lokalita.id)).zfill(7)
-        return prefix + id_number
-    else:
-        logger.error("Could not assign temporary identifier to lokalita with Null ID")
-        logger.error("core.ident_cely.get_temp_lokalita_ident.null_id_error",
-                     extra={"typ": typ, "region": region, "lokalita": lokalita})
-        return None
+    sequence = "{0}".format(str(get_next_sequence("lokalita_xident_seq"))).zfill(9)
+    return prefix + sequence
+
+def get_temp_akce_ident(region):
+    """
+    Metóda pro výpočet dočasného identu samostatný akce.
+
+    Logika složení je: "X-" + region (M anebo C) + "-9" + 9 místne číslo ze sekvence akce_xident_seq doplňeno na 9 číslic -A.
+    
+    Příklad: "X-M-9000123456A"
+    """
+    id_number = "{0}".format(str(get_next_sequence("akce_xident_seq"))).zfill(9)
+    return str(IDENTIFIKATOR_DOCASNY_PREFIX + region + "-9" + id_number + "A")
+
+def get_temp_ez_ident():
+    """
+    Metóda pro výpočet dočasného identu externího zdroje.
+
+    Logika složení je: "X-BIB" + 9 místne číslo ze sekvence externi_zdroj_xident_seq doplňeno na 9 číslic.
+    
+    Příklad: "X-BIB-000123456"
+    """
+    id_number = "{0}".format(str(get_next_sequence("externi_zdroj_xident_seq"))).zfill(9)
```

Comment:
Nepoužívej prosím metodu format, to už je zastaralé. Používej prosím f-stringy.
https://realpython.com/python-f-strings/